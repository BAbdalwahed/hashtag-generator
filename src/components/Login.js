import React from "react";
import { Link } from "react-router-dom";

const Login = () => {
  return (
    <div className="login">
      <h1>#Hashtify</h1>
      <h2>Log in to search, create and manage your hashtags.</h2>
      <button type="button">Log in</button>
      <h2>Or you can search without logging in.</h2>
      <Link to="/search">I just want to search</Link>
    </div>
  );
};

export default Login;
