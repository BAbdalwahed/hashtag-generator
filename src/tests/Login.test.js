import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Login from "../components/Login";

it("renders Login component", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Login />
    </BrowserRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
