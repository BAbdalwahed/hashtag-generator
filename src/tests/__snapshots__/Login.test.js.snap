// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login component 1`] = `
<DocumentFragment>
  <div
    class="login"
  >
    <h1>
      #Hashtify
    </h1>
    <h2>
      Log in to search, create and manage your hashtags.
    </h2>
    <button
      type="button"
    >
      Log in
    </button>
    <h2>
      Or you can search without logging in.
    </h2>
    <a
      href="/search"
    >
      I just want to search
    </a>
  </div>
</DocumentFragment>
`;
